opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -857,3 +845,17 @@ export const convertOperatorToIcon = (operator: string) => {
       return null;
   }
 };
+
+export const extractAllFilters: (filters: FilterGroup) => Filter[] = (filters: FilterGroup) => {
+  const allFilters: Filter[] = [];
+  allFilters.push(...filters.filters);
+  filters.filterGroups.forEach((filterGroup) => extractAllFilters(filterGroup));
+  return allFilters;
+};
+
+export const cleanFeedFilters = (filters: FilterGroup, helpers: handleFilterHelpers, types: string[], completeFilterKeysMap: Map<string, Map<string, FilterDefinition>>) => {
```

Comment:
can be renamed in 'cleanFilters' because the function can be used for other entities than feeds

Reply:
ok